extensions/prototypes/labgraph_monitor/README.md

```diff
@@ -0,0 +1,66 @@
+# LabGraph Monitor
+
+LabGraph Monitor helps visualize the graph built. Inorder to run the application, you will need to first set it up using React using the below steps:
+
+First, incase you do not have npm installed, follow the below steps to get started:
+
+Download Node.js from [here](https://nodejs.org/en/download/) 
+
+Once it is installed, you can verify that node and npm got downloaded successfully by running the following: 
```

Comment:
"Once Node is installed"

Reply:
I'd say "verify that node/npm was installed successfully by running the following:" would be a tad less awkward than the current sentence.